src/main/java/org/jenkinsci/plugins/envinjectapi/util/EnvInjectActionRetriever.java

```diff
@@ -51,7 +51,7 @@ public static Action getEnvInjectAction(@Nonnull Run<?, ?> run) {
             LOGGER.log(Level.WARNING, "There was a problem in the invocation of getParentBuild in hudson.matrix.MatrixRun", e);
         }
 
-        List<Action> actions = run.getActions();
+        List<? extends Action> actions = run.getAllActions();
```

Comment:
I suggest filing a separate pull request for it. It changes the behavior, and it might be a code/bugfix which needs a separate review

Reply:
I moved that to another merge request as suggested.